src/root/session.rs

```diff
@@ -27,6 +27,7 @@ pub struct Session(CF_CONNECT_FLAGS);
 impl Session {
     /// Create a new [Session][crate::Session].
     pub fn new() -> Self {
+        unsafe { CoInitializeEx(std::ptr::null(), COINIT_MULTITHREADED) }.unwrap();
```

Comment:
I'm not too familiar with COM APIs, how does this work and what does it do? 

Does this initialize only the current thread? What if it's called from a different thread? Do we need to add extra checks or remove Send/Sync to ensure it's called on the same thread?

Also, why does it need to be called here?

Reply:
It's weired, when I first time tried to run wincs CfConnectSyncRoot fails and tells me that CoInitialize needs to be called.
Now I tried to delete it but nothing happened like before.

Reply:
Hm, thats weird